client/apps/edit2/components/content/section_controls/index.jsx

```diff
@@ -1,29 +1,39 @@
+// A container for section inputs
+// Position changes on scroll to stick to section top
```

Comment:
This is a small nit, but when documenting at the top of a file its more idiomatic to use multi-line comments, which all editors will create by typing `/**`, and then hit enter. This will expand to 

```javascript
/**
 * A container for section inputs. Position changes on scroll to stick to section top.
 * ... 
 */ 
```


```javascript